scripts/generateDocumentation.ts

```diff
@@ -149,8 +149,12 @@ console.log(markdown);
 
 
 /** True if this is visible outside this file, false otherwise */
-function isNodeExported(node: tsc.Node): boolean {
-    return (tsc.getCombinedModifierFlags(node) & tsc.ModifierFlags.Export) !== 0 || (!!node.parent && node.parent.kind === tsc.SyntaxKind.SourceFile); // tslint:disable-line no-bitwise
+function isNodeExported(node: tsc.Node | tsc.Declaration): boolean {
+    const declaration: tsc.Declaration = {
```

Comment:
This is... annoying. `tsc.getCombinedModifierFlags` now expects a `tsc.Declaration` instead of a `tsc.Node`.
A `Declaration` is a `Node`, except:
```typescript
interface Declaration extends Node {
  _declarationBrand: any;
}
```

This makes some amount of sense: declarations and nodes are distinct concepts in the compiler, so they should have distinct types. However, this type-level change only caused compile-time errors instead of avoiding run-time errors.